lib/psulib_blacklight/solr_manager.rb

```diff
@@ -59,8 +59,13 @@ def initialize_collection
       if next_collection_version == 1
```

Comment:
would the next version ever be greater than 1?

Reply:
yeah, since this gets called at startup, it could be anything